patches/minecraft/net/minecraft/tileentity/IBeehiveTilEntity.java.patch

```diff
@@ -0,0 +1,12 @@
+--- a/net/minecraft/tileentity/IBeehiveTileEntity.java
++++ b/net/minecraft/tileentity/IBeehiveTileEntity.java
+@@ -0,0 +1,9 @@
++package net.minecraft.tileentity;
++
++import net.minecraft.entity.Entity;
++
++public interface IBeehiveTileEntity {
```

Comment:
Forge does not add new interfaces to the `net.minecraft` package. Add it to a Forge package and implement it in the vanilla block

Reply:
Where would be the best place to put this in Forge? net.minecraftforge.common.extensions.IForgeBeehiveTileEntity ?